### PR TITLE
append function

### PR DESCRIPTION
**Pull request by ndurumo254**
fixes https://github.com/unifyai/ivy/issues/11769
fixes https://github.com/unifyai/ivy/issues/11772
fixes https://github.com/unifyai/ivy/issues/11773

### CODE REVIEW

**Comment by ndurumo254** [comment]
https://github.com/unifyai/ivy/issues/11769

**Comment by ndurumo254** [comment]
https://github.com/unifyai/ivy/issues/11772

**Comment by ndurumo254** [comment]
https://github.com/unifyai/ivy/issues/12018

**Comment by ndurumo254** [comment]
https://github.com/unifyai/ivy/issues/12019

File: ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_statistical.py [hunk 1]
```diff
@@ -586,3 +587,91 @@ def test_numpy_nanmin(
...
+
+
+@handle_frontend_test(
+    fn_tree="jax.numpy.nanvar",
+    dtype_x_axis=statistical_dtype_values(function="nanvar"),
+    dtype=helpers.get_dtypes("float", full=False, none=True),
+    where=np_frontend_helpers.where(),
+    keep_dims=st.booleans(),
+)
+def test_numpy_nanvar(
```
Comment (hirwa-nshuti) [inline ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_statistical.py:643]: This should be renamed to `test_jax_numpy_nanvar`

File: ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_statistical.py [hunk 2]
```diff
@@ -586,3 +587,91 @@ def test_numpy_nanmin(
...
+
+
+@handle_frontend_test(
+    fn_tree="jax.numpy.nanstd",
+    dtype_and_a=statistical_dtype_values(function="nanstd"),
+    dtype=helpers.get_dtypes("float", full=False, none=True),
+    where=np_frontend_helpers.where(),
+    keep_dims=st.booleans(),
+)
+def test_numpy_nanstd(
```
Comment (hirwa-nshuti) [inline ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_statistical.py:599]: Should be renamed to `test_jax_numpy_nanstd`

**Review by hirwa-nshuti — CHANGES_REQUESTED** [review]
Hey @ndurumo254 

Sorry for the late review the implementations looks good only two functions with failing tests
* `nanstd`
*  `nanvar`
Fix them I'll merge the PR after their tests are passing😊

Thanks

**Review by hirwa-nshuti — APPROVED** [review]
LGTM!

**Pull request merged**